tasks.py

```diff
@@ -0,0 +1,97 @@
+"""
+PyInvoke developer task file
+https://www.pyinvoke.org/
+
+These tasks can be run using `invoke <NAME>` or `inv <NAME>` from the project root.
+
+To show all available tasks `invoke --list`
+
+To show task help page `invoke <NAME> --help`
+"""
+import invoke
+
+from scripts import check_type_hint_coverage
+
+
+@invoke.task
+def sort(ctx, path=".", check=False, exclude=None):
+    """Sort module imports"""
+    cmds = ["isort", path, "--profile", "black"]
+    if check:
+        cmds.append("--check-only")
+    if exclude:
+        cmds.extend(["--skip", exclude])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task
+def fmt(ctx, path=".", sort_=True, check=False, exclude=None):
+    """Run code formatter"""
+    if sort_:
+        sort(ctx, path, check=check, exclude=exclude)
+
+    cmds = ["black", path]
+    if check:
+        cmds.append("--check")
+    if exclude:
+        cmds.extend(["--exclude", exclude])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task
+def lint(ctx, path="great_expectations/core"):
+    """Run code linter"""
+    ctx.run(" ".join(["flake8", path]))
+
+
+@invoke.task
+def hooks(ctx, changes_only=False, diff=False):
+    """Run pre-commit hooks"""
+    cmds = ["pre-commit", "run"]
+    if diff:
+        cmds.append("--show-diff-on-failure")
+    # used in CI - runs faster and only checks files that have changed
+    if changes_only:
+        cmds.extend(["--from-ref", "origin/HEAD", "--to-ref", "HEAD"])
+    else:
+        cmds.extend(["--all-files"])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task(iterable=["modules"])
+def types(ctx, modules, coverage=False, install_types=False):
+    """Static Type checking"""
+    modules = modules or [
```

Comment:
Exactly.
I plan to address all the errors for all the un-commented modules before merging, and then add the this check to the CI to prevent regressions.

Reply:
Maybe add a comment stating this. They might fall out of date but that's ok.

Reply:
Done. Also I changed it to `packages` instead of `modules`.